src/ontology/cl-edit.owl

```diff
@@ -32238,6 +32248,65 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005385))
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
 
+# Class: obo:CL_4042017 (α1-tanycyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16344112") Annotation(oboInOwl:hasDbXref "PMID:26578855") Annotation(oboInOwl:hasDbXref "https://en.wikipedia.org/wiki/Tanycyte") obo:IAO_0000115 obo:CL_4042017 "A tanycyte cell has its soma located in the ventromedial and dorsomedial nuclei of the hypothalamus. This type of tanycyte extends its protrusions close to parenchymal neurons without contacting blood vessels. It expresses the glial marker S-100β.")
```

Comment:
In the first sentence is missing a conjunction 'that' to link the genus with the differentia: "A tanycyte cell **_that_** has its soma"